lib/ansible/cli/adhoc.py

```diff
@@ -69,7 +69,8 @@ def _play_ds(self, pattern, async_val, poll):
         mytask = {'action': {'module': context.CLIARGS['module_name'], 'args': parse_kv(context.CLIARGS['module_args'], check_raw=check_raw)}}
 
         # avoid adding to tasks that don't support it, unless set, then give user an error
-        if context.CLIARGS['module_name'] not in ('include_role', 'include_tasks') and any(frozenset((async_val, poll))):
+        include_actions = ('include_role', 'include_tasks', 'ansible.builtin.include_role', 'ansible.builtin.include_tasks')
```

Comment:
i would just create a function that appends `ansible.builtin.' prefixed options to provided list

Reply:
I already added such a function ;) I'm now using it everywhere. For efficiency, how about converting all the (now) dynamic function calls to new constants in (f.ex.) constants.py?